src/types/contact-request.ts

```diff
@@ -1,10 +1,10 @@
 /*
-    NOTE: 
+    NOTE:
         I am choosing to use an interface here for simplicity.
         You may choose to create a Type instead and that's okay!
         https://www.typescriptlang.org/play/?#example/types-vs-interfaces
 */
-export interface IContactRequest {
+export interface ContactRequest {
```

Comment:
I'm so, so, so, so onboard with `type` > `interface`, for various reasons.

Avoiding declaration merging is the biggest reason, but type also allows some cool util usages that interface doesn't.

Speed of compilation is the big reason _for_ interface, but - until you need those extra milliseconds per compile, I don't think it's worth.

https://dev.to/treasuredev_/use-type-not-interface-in-typescript-34i7